### PR TITLE
Update Web tiled layer sample with new service

### PR DESCRIPTION
**Pull request by shubham7109**
### Description
PR to update the service used in the `AddWebTiledLayer` sample. Since Stamen will be discontinuing their tiled services, we would need to replace it with an alternative service. 

This PR uses the [World_Ocean_Base](https://server.arcgisonline.com/arcgis/rest/services/Ocean/World_Ocean_Base/MapServer/) service provided by ArcGIS Living Atlas 

### Links and Data
`kotlin/issues/3162`

### CODE REVIEW

File: add-web-tiled-layer/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <resources>
     <string name="app_name">Add web tiled layer</string>
-    <string name="template_uri_stamen">https://stamen-tiles-{subdomain}.a.ssl.fastly.net/terrain/{level}/{col}/{row}.png</string>
-    <string name="stamen_attribution">Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://www.openstreetmap.org/copyright">ODbL</a>.</string>
+    <string name="template_uri_stamen">https://server.arcgisonline.com/arcgis/rest/services/Ocean/World_Ocean_Base/MapServer/tile/{level}/{row}/{col}.jpg</string>
+    <string name="stamen_attribution">Map tiles by <a href="https://livingatlas.arcgis.com">ArcGIS Living Atlas of the World</a>, under <a href="https://www.esri.com/en-us/legal/terms/full-master-agreement">Esri Master License Agreement</a>. Data by Esri, Garmin, GEBCO, NOAA NGDC, and other contributors.</string>
```
Comment (hud10837) [inline add-web-tiled-layer/src/main/res/values/strings.xml:4]: I think these resource string names should be changed now that the sample is not using stamen.

**Review by hud10837 — CHANGES_REQUESTED** [review]
Looks good and works on my device, one change

File: add-web-tiled-layer/src/main/res/values/strings.xml (hunk 1 above)
Comment (shubham7109) [reply add-web-tiled-layer/src/main/res/values/strings.xml]: Good catch, added!

**Comment by shubham7109** [comment]
@hud10837 Thanks for the feedback, ready for your review :)

File: add-web-tiled-layer/README.metadata.json [hunk 2]
```diff
@@ -8,10 +8,7 @@
     ],
     "keywords": [
         "OGC",
-        "Open Street Map",
-        "OpenStreetMap",
         "layer",
-        "stamen.com",
```
Comment (gunt0001) [inline add-web-tiled-layer/README.metadata.json:14]: would it make sense to add "ArcGIS Living Atlas" as a key?

File: add-web-tiled-layer/src/main/java/com/esri/arcgismaps/sample/addwebtiledlayer/MainActivity.kt [hunk 3]
```diff
@@ -39,18 +39,13 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         lifecycle.addObserver(mapView)
 
-        // list of sub-domains
-        val subDomains = listOf("a", "b", "c", "d")
-
         // build the web tiled layer from stamen url and subDomains
```
Comment (gunt0001) [inline add-web-tiled-layer/src/main/java/com/esri/arcgismaps/sample/addwebtiledlayer/MainActivity.kt:42]: This comment needs updating:
```suggestion
        // build the web tiled layer from ArcGIS Living Atlas of the World tile service url and subDomains
```

**Review by gunt0001 — APPROVED** [review]
@shubham7109 - looks good, some minor comments/questions.

**Pull request merged**